reducers/reducers.ts

```diff
@@ -11,6 +11,7 @@ export const CATEGORICAL_VARIABLE_DELETED = 'CATEGORICAL_VARIABLE_DELETED'
 export const CONFIGURATION_UPDATED = 'CONFIGURATION_UPDATED'
 export const RESULT_REGISTERED = 'RESULT_REGISTERED'
 export const DATA_POINTS_ADDED = 'DATA_POINTS_ADDED'
+export const DATA_POINTS_UPDATED = 'DATA_POINTS_EDITED'
```

Comment:
Shouldn't the value match the variable name for DATA_POINTS_UPDATED?

Reply:
Well spotted. Fixed.